pytest_fixtures/rh_cloud.py

```diff
@@ -1,7 +1,25 @@
 import pytest
 from nailgun import entities
 
+from robottelo.config import settings
 from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
+from robottelo.constants import DISTRO_RHEL8
+from robottelo.helpers import add_remote_execution_ssh_key
+from robottelo.rh_cloud_utils import setting_update
+
+
+@pytest.fixture(scope='module')
+def set_rh_cloud_token():
+    """A module-level fixture to set rh cloud token value."""
+    setting_update('rh_cloud_token', str(settings.rh_cloud.token))
```

Comment:
You shouldn't have to wrap the token with `str` here, is this a leftover pattern from NodeSettingsWrapper use?

Reply:
@jameerpathan111 ahhh was this not rebased to a newer version of robottelo? you shouldn't have to deal with the SettingsNodeWrapper

Reply:
updated the pr.